.circleci/config.yml

```
@@ -125,3 +133,19 @@ jobs:
 #          name: Create pull request in api-downloads
 #          command: |
 #            make sdk-registry-publish
+  publish-documentation:
+    environment:
+      - MBX_CI_DOMAIN: o619qyc20d.execute-api.us-east-1.amazonaws.com
+    docker:
+      - image: mbgl/android-ndk-r21e:latest
+    working_directory: ~/code
+    steps:
+      - checkout
+      - setup-aws-credentials
+      - run:
+          name: Publish API docs
+          command: |
+            GITHUB_WRITER_TOKEN=$(./mbx-ci github writer token)
+            git remote set-url origin "https://x-access-token:$GITHUB_WRITER_TOKEN@github.com/mapbox/mapbox-java"
```

Maybe it should be like this?
```suggestion
            git remote set-url origin "https://x-access-token:$GITHUB_WRITER_TOKEN@github.com/mapbox/mapbox-java.git"
```

---

this could be the reason I saw path in logs 🤔

---

no. it works and we have the same ulr in android nav sdk

---

I mean it's better to use the correct HTTPS URL for the repo even though the version without `.git` at the end works somehow.

---

yep, I agree, having right URL it's better.